features/check-if-trending.js

```diff
@@ -0,0 +1,59 @@
+if (window.location.href.startsWith('https://scratch.mit.edu/projects/')) {
+    const waitForShareDate = new MutationObserver(checkTrending);
+    waitForShareDate.observe(document.querySelector('body'), { attributes: true, childList: true, subtree: true });
+    async function checkTrending() {
+        if (document.querySelector('div.share-date') !== null) {
+            waitForShareDate.disconnect()
+        var response = await fetch("https://api.scratch.mit.edu/explore/projects?limit=25&offset=0&language=en&mode=trending&q=*", {
```

Comment:
Maybe change it to 40? we can avoid making two requests. Also, there are redudant headers.

Reply:
Resolved.